modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/Calculator/Calculator.soy

```diff
@@ -134,7 +134,7 @@
 
 	{let $item kind="html"}
 		{$label}
-		<span class="calculate-fieldname">Fieldname: {$title}</span>
+		<span class="calculate-fieldname">Field ID: {$title}</span>
```

Comment:
@brunofariasdeo, instead of use it hard coded, add a new parameter on this template, as `{@param label: string}` for example, populating it with the [proper language key](https://github.com/liferay/liferay-portal/blob/2b52eaf5dc69c23c07d07d1056ae7ec0ae15376e/portal-impl/src/content/Language.properties#L1989). In addition, check this [code](https://github.com/liferay/liferay-portal/blob/2b52eaf5dc69c23c07d07d1056ae7ec0ae15376e/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/Calculator/Calculator.es.js#L102-L104) to get some inspiration. 😉 

Thanks.